src/components/AstroPiModel/AstroPiControls/Input.js

```diff
@@ -9,7 +9,9 @@ const Input = (props) => {
 
   useEffect(() => {
     if (Sk.sense_hat) {
-      Sk.sense_hat[name] = value;
+      startTransition(() => {
+        Sk.sense_hat[name] = value;
+      });
     }
   }, [name, value]);
```

Comment:
I'm not sure what the `startTransition` does here because this is not an animation

Reply:
@loiswells97 Actually, `startTransition` in React 18 is not related to animation; it helps update the UI without rendering blocking. For more information, please visit https://react.dev/reference/react/startTransition 🙌

Reply:
I'd not come across it used outside of animations so thanks for the comments and link 👍